Meta/run.sh

```diff
@@ -54,7 +54,7 @@ installed_major_version=$("$SERENITY_QEMU_BIN" -version | head -n 1 | sed -E 's/
 [ "$installed_major_version" -lt "$SERENITY_QEMU_MIN_REQ_VERSION" ] && die "Required QEMU >= 5.0! Found $($SERENITY_QEMU_BIN -version | head -n 1)"
 
 SERENITY_SCREENS="${SERENITY_SCREENS:-1}"
-if uname -a | grep -iq WSL; then
+if (uname -a | grep -iq WSL) || (uname -a | grep -iq microsoft); then
```

Comment:
Is this change really needed? My WSL2 uname contains WSL already... but maybe that's a change in the more recent WSL kernels? 

```
$ uname -a
Linux LMAO 5.10.16.3-microsoft-standard-WSL2 #1 SMP Fri Apr 2 22:23:49 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

Reply:
I have the latest non-Insider Windows build with WSL2, here is the uname:
```
$ uname -a
Linux GAL-PC 4.19.104-microsoft-standard #1 SMP Wed Feb 19 06:37:35 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

Reply:
Are you sure that's the latest? I've got Windows 10 20H2 Build 19042.1052 and my WSL2 is showing

```
$ uname -a
Linux DESKTOP-QQ5PM6D 5.4.72-microsoft-standard-WSL2 #1 SMP Wed Oct 28 23:40:43 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

Reply:
I have Windows 10 21H1 Build 19043.1081.

Reply:
Looking at the discussion here:
https://github.com/microsoft/WSL/issues/5650

Seems it was setting "Receive updates for other Microsoft Products when you update Windows" to `On` in Windows Update --> Advanced Options that made it download the new kernel? not sure when I did that tbh...

Reply:
That did the trick, thank you!